src/ticketsReducer.ts

```
@@ -0,0 +1,78 @@
+
+    case "addComment": {
+      const targetTicket = tickets.find(
+        (ticket) => ticket.id === action.ticketId
+      );
+      if (!targetTicket) {
+        console.error("Ticket not found");
+        return tickets;
+      }
+      const commentIds = tickets.reduce(
+        (acc, ticket) => [
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)